### PR TITLE
Javascript-javascript2-week2

### PR DESCRIPTION
**Pull request by santhoshboinapally**
javascript2- week2 home work pull request created for review

### CODE REVIEW

File: javascript/javascript2/week2/double.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const numbers = [1, 2, 3, 4];
```
Comment (KavyaReddy732) [inline javascript/javascript2/week2/double.js:1]: i would suggest to add more numbers to the array

File: javascript/javascript2/week2/movies.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+);
+//Calculate the average rating of all the movies using reduce. Optional
+const moviesRateOnly = movies
+    .filter((movie) => movie.rating)
+    .map((movie) => movie.rating);
+const averageRate = moviesRateOnly.reduce((x, y) => {
+    const totalRate = x + y / moviesRateOnly.length;
+    return totalRate;
+}, 0);
+console.log(`The average of the rating of all the movies is ${averageRate}`);
```
Comment (KavyaReddy732) [inline javascript/javascript2/week2/movies.js:72]: very well written 👍

**Review by KavyaReddy732 — COMMENTED** [review]
👍

File: javascript/javascript2/week2/movies.js (hunk 2 above)
Comment (santhoshboinapally) [reply javascript/javascript2/week2/movies.js:72]: Thankyou

**Pull request merged**